### PR TITLE
Pete

### PR DESCRIPTION
**Pull request by petegriffith**
- Added resolver: **updateHouseModules**, tested with /graphql playground
- Added $store action **adminUpdateModulesList**
- Added button to run the above action on the modules chooser view

In order to use the above functionality:

1. Make your changes to the modules.json file
2. (Close the backend server if you've got it running locally)
2. **NPM Run Build** (?)
3. **NPM Run Build:Server**
4. **NPM Run Start**
5. Navigate to the modules section in the drawer and press the button. You may need to refresh the page to see the functionality.

### CODE REVIEW

File: server/index.ts [hunk 1]
```diff
@@ -24,15 +24,15 @@ import sqlConfig from '../ormconfig'
   
   const app = express();
 
-  app.use(express.static(path.resolve(__dirname,"..", "..", "dist")))
+/*   app.use(express.static(path.resolve(__dirname,"..", "..", "dist")))
```
Comment (jaysutham) [inline server/index.ts]: Should un-comment these

File: server/index.ts [hunk 2]
```diff
@@ -24,15 +24,15 @@ import sqlConfig from '../ormconfig'
...
 
   app.use(history({
     disableDotRule: true,
     verbose: true
   }))
 
 
-  app.use(express.static(path.resolve(__dirname,"..", "..", "dist")))
+  app.use(express.static(path.resolve(__dirname,"..", "..", "dist"))) */
```
Comment (jaysutham) [inline server/index.ts]: Should un-comment these

**Review by jaysutham — CHANGES_REQUESTED** [review]

File: server/index.ts (hunk 2 above)
Comment (petegriffith) [reply server/index.ts]: Yep, you're right

File: server/index.ts (hunk 1 above)
Comment (petegriffith) [reply server/index.ts]: Yep

**Review by jaysutham — APPROVED** [review]

**Review by KuroKen91 — APPROVED** [review]
LGTM! Thanks Pete!

**Review by Evomatic — APPROVED** [review]
This looks solid!

**Pull request merged**